package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cloudant",
+  "name": "@cloudant/cloudant",
```

Comment:
It's likely we ought to update the Node engine as well, but perhaps that didn't ought to fall under this change.